src/python/pants/backend/python/lint/pylint/rules_integration_test.py

```diff
@@ -222,12 +226,14 @@ def test_includes_transitive_dependencies(rule_runner: RuleRunner) -> None:
         {
             "BUILD": dedent(
                 """\
-                python_requirement(name='transitive_req', requirements=['fake'])
+                python_requirement(name='transitive_req', requirements=['freezegun'])
```

Comment:
![freeze](https://media.giphy.com/media/e5RRlEV157qHslappH/giphy.gif)

Reply:
hahahah credit to @asherf for teaching me about freezegun

Reply:
Oh hey, that's actually very useful. We have tests that call those functions and are currently passing only because they are cached. :rocket: